server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/methods/MIMEMessageConverter.java

```diff
@@ -220,6 +216,46 @@ private Multipart createMultipart(CreationMessage newMessage, ImmutableList<Mess
         }
     }
 
+    private Multipart createMultipartWithAttachments(CreationMessage newMessage, ImmutableList<MessageAttachment> messageAttachments) throws IOException {
+        MultipartBuilder mixedMultipartBuilder = MultipartBuilder.create(MIXED_SUB_TYPE);
+        Map<Boolean, List<MessageAttachment>> distinguishedAttachments = distinguishAttachments(messageAttachments);
+        List<MessageAttachment> inlineAttachments = Optional.ofNullable(distinguishedAttachments.get(true))
+            .orElse(ImmutableList.of());
+        List<MessageAttachment> normalAttachments = Optional.ofNullable(distinguishedAttachments.get(false))
```

Comment:
Can we find a more specific world than normal?

Reply:
IDK how to call it, from what I see on the internet, it's just `attachments`. The best I can do is `inlines` vs `attachments`

Reply:
besideAttachments?